share/spice/hex/content.handlebars

```diff
@@ -0,0 +1,8 @@
+<p><strong>{{releases.0.version}}</strong> is the latest of <strong>{{releases.length}}</strong> releases and it was published <em>{{updated_at}}</em>.</p>
```

Comment:
We typically avoid bolding text because it's used to indicate exact matches.

You can use the `text--primary`, `text--secondary` classes to help pronounce text.

We have more available at https://duckduckgo.com/styleguide